test/repl/ok/double-import.stdout.ok

```diff
@@ -8,14 +8,12 @@ Motoko compiler (source XXX)
 -- Interpreting empty.mo:
 -- Interpreting stdin:
 -- Finished stdin:
-let anon-import-1.1 : module {...}
```

Comment:
I guess these are gone, because the wildcard pattern has a side-effect-less RHS.

Reply:
I think its gone because we don't have the optional <id>? production which would introduce a gensym-ed, anonymous binding (not a wildcard).